HTML:CSS learning project/index.html

```diff
@@ -23,69 +30,55 @@
-      Neque asperiores tenetur voluptas sed! Laudantium enim voluptates autem veritatis mollitia corrupti itaque quos veniam consectetur voluptatibus id dolorum ut at aliquid, omnis unde sint et aperiam magni vitae ipsam?
+      <div class="row">
+        <div class="col-md">
+          <img src="images/image1.jpg">
+          <h2>Lorem ipsum dolor sit amet.</h2>
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. A, accusamus.</p>
+        </div>
+        <div class="col-md">
+          <img src="images/image2.jpg">
+          <h2>Lorem ipsum dolor sit amet.</h2>
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. A, accusamus.</p>
+        </div>
+        <div class="col-md">
+          <img src="images/image3.jpg">
+          <h2>Lorem ipsum dolor sit amet.</h2>
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. A, accusamus.</p>
+        </div>
+      </div>
+      <div class="row-2">
+        <div class="col-md-2">
+          <img src="images/image3.jpg">
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut, sit! Eveniet libero quia ipsa dolore, exercitationem distinctio nobis officia magnam dicta harum aut natus similique laboriosam! Veniam aliquid maiores magnam facilis at laudantium sequi facere repudiandae officiis impedit, odit qui. Molestiae, similique dolores dicta iusto distinctio culpa voluptate suscipit explicabo voluptatem aliquam ducimus. Nemo et pariatur iste provident libero voluptatem quia perferendis! Ex sequi libero quisquam nihil porro omnis impedit assumenda. Quibusdam, sapiente soluta consequuntur harum, ipsum quas voluptatum atque modi, vero molestias excepturi esse exercitationem nulla beatae deleniti veniam accusamus. Eaque magni accusamus iure quod officia, laborum cum tenetur?</p>
+        </div>
+        <div class="col-md-3">
+          <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Excepturi magni sint labore sapiente recusandae accusamus nulla assumenda sunt ratione maiores animi a, fuga id non cumque delectus expedita deserunt vero, voluptas aliquid nobis temporibus? Similique, praesentium rerum voluptatum accusantium mollitia, animi placeat expedita, dolorum alias iste itaque iure odit totam! Soluta, cum? Minus possimus odio quasi, ipsa optio, maiores similique dolores blanditiis amet corrupti numquam voluptates vel porro quisquam non officiis nihil inventore magni sequi, molestias minima natus. Ducimus alias iure eveniet mollitia numquam quas reprehenderit ut fuga? Dicta, deleniti recusandae ipsum corrupti expedita voluptatibus consequuntur. Commodi debitis soluta tenetur? </p>
+          <img src="images/image3.jpg">
+        </div>
+      </div>
     </div>
   </main>
   <footer class="main-footer">
-    <nav>
-      <a>contact us</a>
-    </nav>
-</footer>
+    <p class="footer-right"><a href="#">Back to top</a></p>
```

Comment:
it isn't a paragraph. If you're using wrapper just for styling purposes, better to stick with div.